src/cpp/server/server.cc

```diff
@@ -258,6 +283,11 @@ bool Server::Start() {
   started_ = true;
   grpc_server_start(server_);
 
+  if (!has_generic_service_) {
+    unknown_method_.reset(new RpcServiceMethod(
+        "unknown", RpcMethod::BIDI_STREAMING, new UnknownMethodHandler));
+    sync_methods_->emplace_back(SyncRequest(unknown_method_.get(), nullptr));
```

Comment:
Why not 

    sync_methods_->emplace_back(unknown_method_.get(), nullptr); 

?